### DISCUSSION TITLE
503 Error after Upgrading to Kimai Version: 1.30.11 stable (prod)

### DISCUSSION THREAD

**Discussion by itrtimrear** [kimai/kimai#3984] [On-Premise: Installation & Updates]
Hi Guys, hi Kevin,

i hope you are doing well!
Last week we have successfully updated some Kimai2 instances to the current stable Versions 1.30.11.
One of this sites is driving me mad. After Updating and clearing the cache, everything is working fine - till you refresh the dashboard for a second time, then it repeatedly comes to error 503. After clearing the cache the dashboard appears for another single time - strange...

Sounds very close like the following topics:
https://github.com/kimai/kimai/discussions/2859
https://github.com/kimai/kimai/discussions/3717
but we are hosting on own servers, not all-ink...

We are using a plesk server (i know kevin you are not a fan of shared hosting, sorry) on PHP 7.4 and 8.1 for testing purpose. There are some more kimai instances on this server running without a problem.

Any idea how we can get rid of this?
I have already backed up the whole datase and installed a freshly new kimai version (1.30.11) which was working like a charm. As long as i imported the database backup which immediately brought back the error. Might there be some problems within the database? Is there a way to scan and fix for this problems? I cant locate informations in server- or kimai- logs regarding this error.

Thank you very much for your help!!!!
regards, Tim

**Comment by kevinpapst** [comment]
Check your PHP modules if you find the ioncube loader there.
Check all logs, a 503 should trigger something somewhere...
A database problem should result in a log entry.

If you have multiple Kimai instances, you can always ask for paid support.

**Reply by itrtimrear** [reply]
Thanks a lot Kevin!

straight after unloading the incubeloader plugin, the 503 error disappeared and everything is working fine.

i have tried to enable the plugin and set the "ioncube.loader.encoded_paths = [pathtolog]" within php.ini and/or .htaccess file of customer without any luck. Do you have any clue? Otherwise i would leave it disabled for a while and try to figure a solution out next weekends.

Anyway, thank you very much for everything. You are doing an amazing job!
